README.md

```diff
@@ -8,92 +8,193 @@
 Metaprogramming tools for DataFrames.jl objects.
 These macros improve performance and provide more convenient syntax.
 
-# Features
+DataFrames.jl has the functions `select`, `transform`, and `combine` 
+for manipulating data frames. DataFramesMeta provides the macros 
+`@select`, `@transform`, and `@combine` to mirror these functions with 
+more convenient syntax. Inspired by [dplyr](https://dplyr.tidyverse.org/) in R 
+and [LINQ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/)
+in C#. 
+
+In addition, DataFramesMeta provides 
+
+* `@orderby`, for sorting data frames
+* `@where`, for keeping rows of a DataFrame matching a given condition
+* `@by`, for grouping and combining a data frame in a single step
+* `@with`, for working with the columns of a data frame with high performance and 
+  convenient syntax
+* `@eachrow`, for looping through rows in data frame, again with high performance and 
+  convenient syntax. 
+* `@linq`, for piping the above macros together, similar to [magrittr](https://cran.r-project.org/web/packages/magrittr/vignettes/magrittr.html)'s
+  `%>%` in R. 
```

Comment:
Suggest to hyperlink each to proper doc. It's pretty hard to understand how these work by reading descriptions. Therefore the descriptions are not that useful in my opinion. Better to list them and link to doc.

Reply:
I think this is a good introduction. The reader of this readme may have never heard of either.